app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>All users in rails</h1>
\ No newline at end of file
+
+<div class="main">
+  <div class="content">
+    <h1>All users in rails</h1>
+    <% @users.each do |user| %>
+      <div class='box flex'>
+        <div class='img flex'>
+          <img src="<%= user.photo %>" alt="unsplash">
+        </div>
+        <div class='user-details'>
+          <p><%= link_to user.name, user_path(user) %></p>
+          <span class="post-no">number of posts: <%= user.posts_counter %></span>
+        </div>
+      </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `partial` file for one user card for best practices. I suggest you refer this **[article](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)** to help you how to do it and more details about partail files.